core/src/main/java/org/opensearch/sql/analysis/DataSourceSchemaIdentifierNameResolver.java

```diff
@@ -35,7 +34,7 @@ public DataSourceSchemaIdentifierNameResolver(
       DataSourceService dataSourceService, List<String> parts) {
     this.dataSourceService = dataSourceService;
     List<String> remainingParts = captureSchemaName(captureDataSourceName(parts));
-    identifierName = String.join(DOT, remainingParts);
+    identifierName = String.join(PathUtils.SEPARATOR, remainingParts);
```

Comment:
you can static import the SEPARATOR
```suggestion
    identifierName = String.join(SEPARATOR, remainingParts);
```

Reply:
Done ✅